### PR TITLE
Fix `make deploy`

### PR DESCRIPTION
**Pull request by fabianvf**
also make deploy accept all keys so it's configurable for up and downstream.

Haven't had a chance to test yet due to busted environment, but verified the generated openshift yaml file looked correct.

### CODE REVIEW

**Comment by jwmatthews** [comment]
What if we changed 'make deploy' to use the same template we use for deploying elsewhere

https://github.com/openshift/ansible-service-broker/blob/master/templates/deploy-ansible-service-broker.template.yaml

File: templates/broker-deployment_template.yaml [hunk 1]
```diff
@@ -18,16 +18,52 @@ spec:
     spec:
       restartPolicy: Always
       containers:
-        - image: "{{dockerhub_org}}/ansible-service-broker:latest"
+        - image: {{registry_org}}/ansible-service-broker:latest
```
Comment (shawn-hurley) [inline templates/broker-deployment_template.yaml]: What if I want to pull ansible-service-broker from my org but get the apbs from ansibleplaybookbundle? I think re-using registry_org might make that hard?

Comment (eriknelson) [inline templates/broker-deployment_template.yaml]: +1, I was frequently doing this.

**Comment by fabianvf** [comment]
@eriknelson comments addressed.

**Comment by eriknelson** [comment]
Thanks @fabianvf, will review and try to get this merged today.

**Comment by fabianvf** [comment]
Going with a completely different approach, these changes are no longer useful.

**Comment by jmrodri** [comment]
Please do not work on deploy. I'm working on card that will update it.

**Pull request closed without merging**